requirements-jenkins-job-builder.txt

```diff
@@ -0,0 +1,3 @@
+python-jenkins==0.4.15  # newer ones have broken basic auth :(
```

Comment:
we can't add these to `requirements.txt` can we? I'm pretty sure I figured out there was a good reason in the last PR if you can jog my memory 😁

Reply:
I think the reason was because we want to use `requirements-jenkins-job-builder.txt` on the Jenkins box, and if we add it to `requirements.txt` then we will end up installing Ansible on the Jenkins box. That might not be an issue though.